cli/cmd/context/ls.go

```diff
@@ -93,7 +94,7 @@ func runList(cmd *cobra.Command, opts lsOpts) error {
 		return nil
 	}
 
-	if opts.json {
+	if format == formatter.JSON || format == formatter.TemplateJSON {
```

Comment:
Looks like it doesn't work when passing `--json`. Could you please add a test for that?

Reply:
done